src/main/java/seedu/address/logic/commands/UntagEventCommand.java

```diff
@@ -30,7 +30,7 @@ public class UntagEventCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_PERSONS + "1 3 4";
     public static final String MESSAGE_UNTAG_EVENT_SUCCESS = "Untagged Persons: %s to Event: %s";
-    public static final String MESSAGE_INVALID_PERSON = "This person: %d. %s doesn't exist in the event";
+    public static final String MESSAGE_INVALID_PERSON = "This person: %d. %s is not tagged to the event";
```

Comment:
It might be better to add the event title in the error message as well, as suggested by the tester
```suggestion
    public static final String MESSAGE_INVALID_PERSON = "This person: %d. %s is not tagged to the event: %s";
```